config/samples/cucp_config.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: ref.nephio.org/v1alpha1
+kind: Config
+metadata:
+  name: cucp-gnb-edge01
+  namespace: oai-ran
+spec:
+  config:
+    apiVersion: workload.nephio.org/v1alpha1
+    kind: NFDeployment
+    metadata: 
+      name: oai-ran-cu-cp
+      namespace: oai-ran
+    spec:
+      provider: oai-cucp.nephio.org
```

Comment:
@josephthaliath for OAI can we use the same provider? I will use `{NF_TYPE}.openairinterface.org`. For example, for UPF it will be --> `upf.openairinterface.org`.